src/System.Private.ServiceModel/src/Internals/WcfEventSource.cs

```diff
@@ -12,6 +12,38 @@ internal sealed class WcfEventSource : EventSource
     {
         public static WcfEventSource Instance = new WcfEventSource();
 
+        // The following four dummy events are to set the correct values for the eventlog channel keyword values. EventSource allocates
+        // the channel id's in the order that they are first used in the event methods lexically. So the first method attributed with [Event]
+		// which specifies an EventChannel will have the first channel keyword assigned to that Channel. We need EventSource to assign
+		// keyword bit 63 to Admin, bit 62 to Operational, bit 61 to Analytic and bit 60 to Debug.
+        [Event(EventIds.DummyAdminEventId, Level = EventLevel.Critical, Channel = EventChannel.Admin, Opcode = EventOpcode.Info, Task = EventTask.None,
+    Keywords = Keywords.DummyKeyword)]
```

Comment:
Nit: (non-blocking) -- the indenting of the new [Event]s doesn't match the others